templates/pages/login.html.twig

```diff
@@ -34,97 +34,94 @@
-            {% endif %}
+                {% if config('display_login_source') %}
+                    <div class="mb-3">
+                        <label class="form-label" for="dropdown_auth{{ rand }}">{{ __('Login source') }}</label>
+                        {{ auth_dropdown_login|raw }}
+                    </div>
+                {% endif %}
 
-            {% if config('login_remember_time') %}
-               <div class="mb-2">
-                  <label class="form-check" for="login_remember">
-                     <input type="checkbox" class="form-check-input" id="login_remember" name="{{ rmbfield }}" {{ config('login_remember_default') ? 'checked' : '' }} />
-                     <span class="form-check-label">{{ __('Remember me') }}</span>
-                  </label>
-               </div>
-            {% endif %}
+                {% if config('login_remember_time') %}
+                    <div class="mb-2">
+                        <label class="form-check" for="login_remember">
+                            <input type="checkbox" class="form-check-input" id="login_remember" name="{{ rmbfield }}" {{ config('login_remember_default') ? 'checked' : '' }}/>
+                            <span class="form-check-label">{{ __('Remember me') }}</span>
+                        </label>
+                    </div>
+                {% endif %}
 
-            <div class="form-footer">
-               <button type="submit" name="submit" class="btn btn-primary w-100" tabindex="3">
-                  {{ __('Sign in') }}
-               </button>
+                <div class="form-footer">
+                    <button type="submit" name="submit" class="btn btn-primary w-100" tabindex="3">
+                        {{ __('Sign in') }}
+                    </button>
+                </div>
             </div>
-         </div>
-
-         {% if right_panel %}
-            <div class="col-auto px-2 text-center">
-               {% if text_login|length > 0 %}
-                  <div class="rich_text_container">
-                     {{ text_login|safe_html }}
-                  </div>
-               {% endif %}
-
-               {% if config('use_public_faq') %}
-                  <hr />
 
-                  <a class="btn btn-outline-secondary" href="front/helpdesk.faq.php">
-                     <i class="fas fa-question"></i>&nbsp;
-                     {{ __('FAQ') }}
-                  </a>
-               {% endif %}
-
-               {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::DISPLAY_LOGIN')) }}
+            {% if right_panel %}
+                <div class="col-auto px-2 text-center">
+                    {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::DISPLAY_LOGIN')) }}
+                </div>
+            {% endif %}
+        </div>
+        {% if config('use_public_faq') %}
+            <div class="text-center mt-4 border-top">
+                <a class="btn btn-outline-secondary mt-4" href="{{ path(front/helpdesk.faq.php) }}">
```

Comment:
Shouldn't the supplied path be a string ?

Reply:
Fixed.